### PR TITLE
fix hugegraph-hbase and hugegraph-example checkstyle issue

### PR DESCRIPTION
**Pull request by mayelena**

### CODE REVIEW

File: hugegraph-hbase/src/main/java/com/baidu/hugegraph/backend/store/hbase/HbaseTable.java [hunk 1]
```diff
@@ -283,7 +283,8 @@ protected BackendEntryIterator newEntryIterator(Query query,
         });
     }
 
-    protected void parseRowColumns(Result row, BackendEntry entry, Query query, boolean enablePartition)
+    protected void parseRowColumns(Result row, BackendEntry entry, Query query,
+                                    boolean enablePartition)
```
Comment (javeme) [inline hugegraph-hbase/src/main/java/com/baidu/hugegraph/backend/store/hbase/HbaseTable.java:287]: align with "Result"

File: hugegraph-hbase/src/main/java/com/baidu/hugegraph/backend/store/hbase/HbaseStore.java [hunk 2]
```diff
@@ -54,7 +54,7 @@ public abstract class HbaseStore extends AbstractBackendStore<Session> {
 
     private static final Logger LOG = Log.logger(HbaseStore.class);
 
-    private final BackendFeatures FEATURES;
+    private final BackendFeatures features;
```
Comment (javeme) [inline hugegraph-hbase/src/main/java/com/baidu/hugegraph/backend/store/hbase/HbaseStore.java:57]: also update references

File: hugegraph-hbase/src/main/java/com/baidu/hugegraph/backend/store/hbase/HbaseSessions.java [hunk 3]
```diff
@@ -84,6 +86,7 @@
 import com.google.common.util.concurrent.Futures;
 
 public class HbaseSessions extends BackendSessionPool {
+    private static final Logger LOG = Log.logger(HbaseSessions.class);
```
Comment (javeme) [inline hugegraph-hbase/src/main/java/com/baidu/hugegraph/backend/store/hbase/HbaseSessions.java:89]: add a blank line after class define

File: hugegraph-hbase/src/main/java/com/baidu/hugegraph/backend/store/hbase/HbaseSessions.java [hunk 4]
```diff
@@ -222,19 +225,21 @@ public void createTable(String table, List<byte[]> cfs) throws IOException {
...
+    public void createPreSplitTable(String table, List<byte[]> cfs, 
+                                    short numOfPartitions) throws IOException {
         TableDescriptorBuilder builder = TableDescriptorBuilder.newBuilder(
                 TableName.valueOf(this.namespace, table));
         for (byte[] cf : cfs) {
             builder.setColumnFamily(ColumnFamilyDescriptorBuilder.newBuilder(cf)
                     .build());
         }
-        byte[][] splits = new byte[numOfPartitions - 1][org.apache.hadoop.hbase.util.Bytes.SIZEOF_SHORT];
+        byte[][] splits = new byte[numOfPartitions - 1][org.apache.hadoop
```
Comment (javeme) [inline hugegraph-hbase/src/main/java/com/baidu/hugegraph/backend/store/hbase/HbaseSessions.java:241]: prefer to wrap line before "[org"

File: hugegraph-hbase/src/main/java/com/baidu/hugegraph/backend/store/hbase/HbaseSessions.java [hunk 5]
```diff
@@ -793,18 +798,18 @@ public long storeSize(String table) throws IOException {
          */
         @SuppressWarnings("unused")
         private void dump(String table, Scan scan) throws IOException {
-            System.out.println(String.format(">>>> scan table %s with %s",
+            LOG.info(String.format(">>>> scan table %s with %s",
```
Comment (javeme) [inline hugegraph-hbase/src/main/java/com/baidu/hugegraph/backend/store/hbase/HbaseSessions.java:801]: LOG.info(">>>> scan table {} with {}",

File: hugegraph-hbase/src/main/java/com/baidu/hugegraph/backend/store/hbase/HbaseSessions.java [hunk 6]
```diff
@@ -793,18 +798,18 @@ public long storeSize(String table) throws IOException {
...
                 Result row = iterator.next();
-                System.out.println(StringEncoding.format(row.getRow()));
+                LOG.info(StringEncoding.format(row.getRow()));
                 CellScanner cellScanner = row.cellScanner();
                 while (cellScanner.advance()) {
                     Cell cell = cellScanner.current();
                     byte[] key = CellUtil.cloneQualifier(cell);
                     byte[] val = CellUtil.cloneValue(cell);
-                    System.out.println(String.format("  %s=%s",
+                    LOG.info(String.format("  %s=%s",
```
Comment (javeme) [inline hugegraph-hbase/src/main/java/com/baidu/hugegraph/backend/store/hbase/HbaseSessions.java:812]: ditto

File: hugegraph-hbase/src/main/java/com/baidu/hugegraph/backend/store/hbase/HbaseStore.java [hunk 7]
```diff
@@ -445,7 +447,8 @@ public static class HbaseSchemaStore extends HbaseStore {
 
         public HbaseSchemaStore(HugeConfig config, BackendStoreProvider provider,
                                 String namespace, String store) {
-            super(provider, namespace, store, config.get(HbaseOptions.HBASE_ENABLE_PARTITION).booleanValue());
+            super(provider, namespace, store, config.get(HbaseOptions
```
Comment (javeme) [inline hugegraph-hbase/src/main/java/com/baidu/hugegraph/backend/store/hbase/HbaseStore.java:450]: wrap line before config

File: hugegraph-hbase/src/main/java/com/baidu/hugegraph/backend/store/hbase/HbaseStore.java [hunk 8]
```diff
@@ -492,7 +495,8 @@ public static class HbaseGraphStore extends HbaseStore {
         private boolean enablePartition;
         public HbaseGraphStore(HugeConfig config, BackendStoreProvider provider,
                                String namespace, String store) {
-            super(provider, namespace, store, config.get(HbaseOptions.HBASE_ENABLE_PARTITION).booleanValue());
+            super(provider, namespace, store, config.get(HbaseOptions.HBASE_ENABLE_PARTITION)
```
Comment (javeme) [inline hugegraph-hbase/src/main/java/com/baidu/hugegraph/backend/store/hbase/HbaseStore.java:498]: wrap line before config

File: hugegraph-example/src/main/java/com/baidu/hugegraph/example/PerfExampleBase.java [hunk 9]
```diff
@@ -133,8 +133,8 @@ public void testQueryVertexPerf(GraphManager graph,
         }, threadCount);
 
         final long size = (PERSON_NUM + SOFTWARE_NUM) * threadCount * times;
-        LOG.info("Query rate with threads: {} vertices/s, " +
-                 "query total vertices {}, cost time: {}ms",
+        LOG.info("Query rate with threads: {} vertices/s,
+                 query total vertices {}, cost time: {}ms",
```
Comment (javeme) [inline hugegraph-example/src/main/java/com/baidu/hugegraph/example/PerfExampleBase.java:137]: ditto

File: hugegraph-example/src/main/java/com/baidu/hugegraph/example/GraphOfTheMoviesExample.java [hunk 10]
```diff
@@ -44,31 +44,31 @@ public static void main(String[] args) {
...
-        System.out.println(">>>> query all edges: size=" + edges.toList().size());
+        LOG.info(">>>> query all edges: size= {}", edges.toList().size());
 
         // query edges by condition
         List<Edge> tomhanksMovies = graph.traversal().V()
                                          .hasLabel("person")
                                          .has("name", "Tom Hanks")
                                          .outE("ACTED_IN").toList();
-        System.out.println(">>>> Tom Hanks ACTED_IN: " + tomhanksMovies);
+        LOG.info{(">>>> Tom Hanks ACTED_IN: {}", tomhanksMovies);
```
Comment (javeme) [inline hugegraph-example/src/main/java/com/baidu/hugegraph/example/GraphOfTheMoviesExample.java:71]: remove "{"

File: hugegraph-example/src/main/java/com/baidu/hugegraph/example/Example3.java [hunk 11]
```diff
@@ -82,44 +82,44 @@ public static void loadNeighborRankData(final HugeGraph graph) {
...
 
-        E.addEdge("directedBy", K);
-        F.addEdge("directedBy", B);
-        F.addEdge("directedBy", L);
+        e.addEdge("directedBy", k);
+        f.addEdge("directedBy", b);
+        f.addEdge("directedBy", l);
 
-        G.addEdge("directedBy", M);
+        G.addEdge("directedBy", m);
```
Comment (javeme) [inline hugegraph-example/src/main/java/com/baidu/hugegraph/example/Example3.java:122]: Error:  /home/runner/work/incubator-hugegraph/incubator-hugegraph/hugegraph-example/src/main/java/com/baidu/hugegraph/example/Example3.java:[122,9] cannot find symbol
  symbol:   variable G
  location: class com.baidu.hugegraph.example.Example3

**Review by javeme — DISMISSED** [review]

File: hugegraph-hbase/src/main/java/com/baidu/hugegraph/backend/store/hbase/HbaseSessions.java [hunk 12]
```diff
@@ -222,19 +226,21 @@ public void createTable(String table, List<byte[]> cfs) throws IOException {
...
     }
 
-    public void createPreSplitTable(String table, List<byte[]> cfs, short numOfPartitions) throws IOException {
+    public void createPreSplitTable(String table, List<byte[]> cfs, 
+                                    short numOfPartitions) throws IOException {
         TableDescriptorBuilder builder = TableDescriptorBuilder.newBuilder(
                 TableName.valueOf(this.namespace, table));
         for (byte[] cf : cfs) {
             builder.setColumnFamily(ColumnFamilyDescriptorBuilder.newBuilder(cf)
                     .build());
```
Comment (imbajin) [inline hugegraph-hbase/src/main/java/com/baidu/hugegraph/backend/store/hbase/HbaseSessions.java:240]: align or not to divide?

File: hugegraph-hbase/src/main/java/com/baidu/hugegraph/backend/store/hbase/HbaseSessions.java [hunk 13]
```diff
@@ -222,19 +226,21 @@ public void createTable(String table, List<byte[]> cfs) throws IOException {
...
         }
-        byte[][] splits = new byte[numOfPartitions - 1][org.apache.hadoop.hbase.util.Bytes.SIZEOF_SHORT];
+        byte[][] splits = new byte[numOfPartitions - 1]
+                                  [org.apache.hadoop.hbase.util.Bytes.SIZEOF_SHORT];
```
Comment (imbajin) [inline hugegraph-hbase/src/main/java/com/baidu/hugegraph/backend/store/hbase/HbaseSessions.java:243]: seems a little strange for readable

**Review by imbajin — COMMENTED** [review]
@seagle-yuan if we don't divide these case, will it hit against the check-style?

File: hugegraph-hbase/src/main/java/com/baidu/hugegraph/backend/store/hbase/HbaseSessions.java [hunk 14]
```diff
@@ -793,18 +798,17 @@ public long storeSize(String table) throws IOException {
...
                 Result row = iterator.next();
-                System.out.println(StringEncoding.format(row.getRow()));
+                LOG.info(StringEncoding.format(row.getRow()));
                 CellScanner cellScanner = row.cellScanner();
                 while (cellScanner.advance()) {
                     Cell cell = cellScanner.current();
                     byte[] key = CellUtil.cloneQualifier(cell);
                     byte[] val = CellUtil.cloneValue(cell);
-                    System.out.println(String.format("  %s=%s",
+                    LOG.info(String.format("  {}={}",
```
Comment (javeme) [inline hugegraph-hbase/src/main/java/com/baidu/hugegraph/backend/store/hbase/HbaseSessions.java:811]: remove String.format and update alignment

**Review by javeme — APPROVED** [review]

File: hugegraph-hbase/src/main/java/com/baidu/hugegraph/backend/store/hbase/HbaseSessions.java (hunk 12 above)
Comment (seagle-yuan) [reply hugegraph-hbase/src/main/java/com/baidu/hugegraph/backend/store/hbase/HbaseSessions.java]: No rules will be hit.
The below line is indented more than 4 blankspace. 
And we setted forceStrictCondition = false, so more than 4 space is ok.

https://checkstyle.sourceforge.io/config_misc.html#Indentation.

there isn't any rules about align in all checkstyle rules

https://checkstyle.sourceforge.io/checks.html

File: hugegraph-hbase/src/main/java/com/baidu/hugegraph/backend/store/hbase/HbaseSessions.java (hunk 13 above)
Comment (seagle-yuan) [reply hugegraph-hbase/src/main/java/com/baidu/hugegraph/backend/store/hbase/HbaseSessions.java:242]: it rely on the setting of IDE

**Review by imbajin — APPROVED** [review]

**Pull request merged**